Github action to copy to sourceforge does not work

https://github.com/checkstyle/checkstyle/actions/runs/3813517044/jobs/6487301635
```
Run ./.ci/copy-site-to-sourceforge.sh 10.6.0
  ./.ci/copy-site-to-sourceforge.sh 10.6.0
  shell: /usr/bin/bash -e {0}
  env:
    SF_USER: ***
RELEASE version:10.6.0
Creating shell for ***@shell.sourceforge.net
Pseudo-terminal will not be allocated because stdin is not a terminal.
sign_and_send_pubkey: no mutual signature supported
***@shell.sourceforge.net: Permission denied (publickey,keyboard-interactive).
Error: Process completed with exit code 255.
```

during execution on local it does demand from me password in interactive mode. .... somethig is missed on my local setup
after shh creation phase it waited for "exit" to continue shell execution

--- a/.ci/copy-site-to-sourceforge.sh
+++ b/.ci/copy-site-to-sourceforge.sh
@@ -16,7 +16,7 @@ fi
 checkForVariable "SF_USER"
 
 echo "Creating shell for $SF_USER@shell.sourceforge.net"
-ssh -i ~/.ssh/private_sourceforge_key -t "$SF_USER"@shell.sourceforge.net create
+ssh -i ~/.ssh/private_sourceforge_key "$SF_USER"@shell.sourceforge.net create
 
 echo "Creating .ci-temp if it does not exist"
 mkdir -p .ci-temp
@@ -35,10 +35,10 @@ tar cfz checkstyle.github.io.tar.gz checkstyle.github.io
 
 echo "Uploading to sourceforge"
 scp -i ~/.ssh/private_sourceforge_key checkstyle.github.io.tar.gz \
-  "$SF_USER"@shell.sourceforge.net:/home/project-web/checkstyle
+  "$SF_USER"@web.sourceforge.net:/home/project-web/checkstyle
 
 echo "Using shell for $SF_USER@shell.sourceforge.net"
-ssh -i ~/.ssh/private_sourceforge_key -t "$SF_USER"@shell.sourceforge.net << 'EOF'
+ssh -i ~/.ssh/private_sourceforge_key "$SF_USER"@shell.sourceforge.net << 'EOF'
 
 cd /home/project-web/checkstyle
 

--- a/.github/workflows/copy_site_to_sourceforge.yml
+++ b/.github/workflows/copy_site_to_sourceforge.yml
@@ -23,13 +23,16 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Configure SSH
+        env:
+          SF_SSH_KEY: ${{ secrets.SF_SSH_KEY }}
         run: |
           mkdir -p ~/.ssh/
           eval "$(ssh-agent -s)"
-          echo "${{ secrets.SF_SSH_KEY }}" > ~/.ssh/private_sourceforge_key
+          echo "$SF_SSH_KEY" > ~/.ssh/private_sourceforge_key
           chmod 400 ~/.ssh/private_sourceforge_key
           ssh-add ~/.ssh/private_sourceforge_key
           ssh-keyscan -t ed25519 shell.sourceforge.net >> ~/.ssh/known_hosts
+          ssh-keyscan -t ed25519 web.sourceforge.net >> ~/.ssh/known_hosts
 
       - name: Run Shell Script
         env:

